concordia/static/scss/action-app.scss

```diff
@@ -82,11 +89,19 @@ main {
     }
 
     .asset-tooltip {
-        background-color: #000;
+        margin: -5px;
```

Comment:
This is cancelling out the padding on `.asset`, so perhaps we should make a variable in this block which we could use there and `#{-1 * …}` here, double it for the width/height calculations below, etc.